media-performance/problem-statement.md

```diff
@@ -0,0 +1,42 @@
+Web Performance on Embedded Devices
+==============================================
+
+Background
+----------
+
+Building a performant and polished UI for a TV application using HTML is extremely difficult. While devices have steadily increased their processing power, the CPU is heavily taxed by Javascript and HTML rendering. The HTML rendering has only gotten worse as users expect a polished interface with gradients, drop shadows, animations and performance of a native app. However, the more complicated the UI the less time there is to use the main thread to do API / business logic work. In order to achieve 60fps, there is 16ms of processing time per frame and the browser will use most of that to do housekeeping work. Even without any business logic, the UI is regularly janky and with business logic and API calls and lazy loading data, the performance of the UI is too poor to ship to customers.
```

Comment:
Thanks @chiefcll. The only suggestion I have would be to rephrase the Background slightly to be not so critical, but still highlight the problem.

```suggestion
Building a performant and polished UI for a TV application using HTML is extremely difficult. While devices have steadily increased their processing power, the CPU is heavily taxed by JavaScript and HTML rendering. The performance of HTML rendering is no longer enough to meet users expectations for a polished interface with gradients, drop shadows, animations and performance of a native app. The more complicated the UI the less time there is to use the main thread to do API and business logic work. In order to achieve 60fps, there is 16ms of processing time per frame and the browser will use most of that to do housekeeping work. Even without any business logic, the UI is regularly janky and with business logic and API calls and lazy loading data, the performance of the UI is compromised.
```